BraveWallet/Crypto/Stores/TransactionConfirmationStore.swift

```diff
@@ -203,6 +203,25 @@ public class TransactionConfirmationStore: ObservableObject {
     }
   }
   
+  private func fetchTransactions(completion: (() -> Void)? = nil) {
```

Comment:
No reason to have this closure optional & nil since its not used with nil anywhere and its a private method